scss/_patterns_navigation.scss

```diff
@@ -190,6 +190,20 @@ $navigation-height: calc(map-get($settings-text-p, line-height) + 2 * $spv--larg
     }
   }
 
+  .p-navigation--sliding {
+    @property --right {
```

Comment:
Is there a reason to define those using full `@property` notation, instead of just defining their initial value:

```scss
--right: 0%;
--up: 0%;
```?

Also, even though they are local variables inside navigation, it could be a good practice to prefix them in similar way other Vanilla CSS variables are prefixed, something like `--vf-nav-sliding-right`.